pegasus/sites.v3/hourofcode.com/public/resources/index.moved

```diff
@@ -1 +1 @@
-/resources/promote
+/promote
```

Comment:
Originally it went to /resources which would have made it go to the "How to get involved" list, which is now /promote. I think that we made it /resources/promote before because we didn't have a good alternative. I thought this was best, but let me know if you think we should change it.